flexget/plugins/notifiers/slack.py

```diff
@@ -35,7 +36,74 @@ class SlackNotifier(object):
             'channel': {'type': 'string'},
             'username': {'type': 'string', 'default': 'Flexget'},
             'icon_emoji': {'type': 'string'},
-            'icon_url': {'type': 'string', 'format': 'url'}
+            'icon_url': {'type': 'string', 'format': 'url'},
+            'attachments': {
```

Comment:
I just read over the documentation, it doesn't look like any of the fields are mandatory?  This is shown in my example config which is actually two attachments (first being the picture, second being the content).  

I'm also not sure what the benefit would be in setting defaults.  My example config relies heavily on `imdb_lookup` but I can imagine we can't assume people are using that plugin let alone want to use attachments instead of a basic message.